### PR TITLE
feat: add manifest example yaml file

### PR DESCRIPTION
**Pull request by nitishfy**
- Add the manifest example to highlight how the example manifest will look like for OBS

### CODE REVIEW

File: example.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      - userid: cpanato
+        role: bugowner
+      - userid: jrrickard
+        role: bugowner
+      - userid: xmudrii
+        role: bugowner
+    repositories:
+      - name:
+        architectures:
+          -
```
Comment (cpanato) [inline example.yaml]: Is this expected?

File: example.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          project:
+          package:
+    subprojects:
+      - name: "isv:kubernetes:addons"
+        rootProject: "isv:kubernetes"
+        title: Addons for Kubernetes Clusters
+        description: Various projects that are used in addition to the core packages hosted in "isv:kubernetes:core"
+        url:
+        persons:
+          - userid: sagrunert
```
Comment (cpanato) [inline example.yaml]: I wpuld add fake usernames

File: example.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                          - aarch64
+                          - ppc64le
+                          - s390x
+                        path:
+                          - project: Ubuntu:20.04
+                            repository: universe
+                          - project: Ubuntu:debbuild
+                            repository: Ubuntu_20.04
+                    build:
+                      disable: {}
```
Comment (cpanato) [inline example.yaml]: Nit: new line

**Review by cpanato — CHANGES_REQUESTED** [review]

File: example.yaml (hunk 2 above)
Comment (nitishfy) [reply example.yaml]: I thought of creating a real manifest that would match how our actual manifest content would look like.

File: example.yaml (hunk 1 above)
Comment (nitishfy) [reply example.yaml]: ah no, these can be omitted.

File: example.yaml (hunk 3 above)
Comment (nitishfy) [reply example.yaml]: done

**Pull request closed without merging**